docs/docs/bridges/doctrine-dbal.rst

```diff
@@ -1,7 +1,7 @@
 Bridge with ``doctrine/dbal``
 ============================================================
 
-See `official documentation <https://www.doctrine-project.org/projects/dbal.html>`__ on Doctrine's website.
+Refer to the official documentation <https://www.doctrine-project.org/projects/dbal.html>__ on Doctrine's website.
```

Comment:
"Refer to the" is fine, but please do not break rst format
I'm still struggling with it so...

Reply:
This comment applies for the whole PR (because there's many like this)